portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,56 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <title>Dan Le's Portfolio</title>      
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script defer src="script.js"></script>
+    <script defer src="intro-script.js"></script>
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles.css">
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body >
+    <main id="swup" class="transition-fade">
+    <div id="bg-image">
+      <div class="container">
+        <div class="header">
+                </nav>
+                <div class="title">
+                    <h2 style="position: absolute;
+                                left: 25%;
+                                top: 5%;
+                                transform: translate(-50%, -50%);
+                                color: #fff;
+                                font-size: 30px;
+                                font-family: 'Abel', sans-serif;
+                                ">Welcome to...</h2>
+                    <h1 style= "border-top: 3px solid #fff; border-bottom: 3px solid #fff;">DAN HIEU LE</h1>
+                    <h2 style="position: absolute;
+                                top: 60%;
+                                left: 68%;
+                                color: #fff;
+                                font-size: 30px;
+                                font-family: 'Abel', sans-serif;
+                                "> portfolio</h2>
+                </div>
+                <button onclick="getRandomImageAsync()"
+                        style="position: absolute;
+                        top: 90%;
+                        left:25%;">Click here to change the background with a random image from the server</button>
```

Comment:
Nit: A shorter button might be good, something like "Click here to change the background!" "Different Background" might be nice.